### ISSUE TITLE
Many language packs installed by default

### ISSUE THREAD

**Issue by joey04** [pop-os/pop#90]
While installing the latest iso a little while ago I noticed that it fetched many language packs.
Now configuring Firefox, I see the many language packs in the Addons view. 

Nothing is misconfigured, so no bugs. But I'm wondering if there could be an option to install these language packs in the first place. (Perhaps when someone buys a Sys76 machine specific pack(s) would be installed by you with their stated language preference and/or geographical location?)

For context, I've tried a number of distros in recent years and don't recall seeing this, though I do realize as a PC vendor you're a unique distro provider. (And I like the fact that you're doing this, and so far Pop is nice overall.)

**Comment by jackpot51** [comment]
This was intentional - to allow offline selection of many languages after installing Pop.

If you want to configure languages, you can build an iso from the https://github.com/pop-os/iso repository, and modify the mk/languages.mk file.

**Comment by joey04** [comment]
I verified all unwanted language packs can be deleted in Synaptic. Now Firefox no longer has a language pack option listed on the Addons page. Also removed them for Gnome, LibreOffice, hunspell.

**Comment by joey04** [comment]
One more thing to mention: in Firefox before I removed them the Addons list also had packs for British and South African English. They were disabled because of being incompatible with version 57. I didn't see either of them listed in Synaptic when I removed the others, so perhaps this is an artefact of Canonical Firefox. (I also have an Ubuntu 17.10 install in a VM and its Firefox has those 2 language packs installed by default.)
